html/signupsignin.html

```diff
@@ -52,6 +46,26 @@ <h2>Sign in</h2>
             </form>
         </div>
     </div>
+
+    <script src="signupsignin.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/7.15.1/firebase-app.js"></script>
```

Comment:
according to the documentation, this imports should be before you'll use them. So please move them up, together with firebaseConfig and initialization.

Reply:
Moved them up.
However I tested it and it's still not adding users into console in firebase.
Chrome console is not giving me any bugs either, so a bit lost...